R/add_lines.R

```diff
@@ -0,0 +1,25 @@
+#' Add lines
+#'
+#' @param data dataset to divide into lines
+#' @param extract_length total length of extract in ms
+#' @param window_size length of line in ms
+#'
+#' @return data set with 'line' column added
+#' @export
+#'
+add_lines <- function(data,extract_length=600000,window_size=60000) {
```

Comment:
```suggestion
add_lines <- function(data,extract_length=600000,window_size=60000) {
```

Would it be an option to change the arguments so that a user can indicate (1) the total length (in minutes) of the fragment they want to visualize, and (2) the number of lines?

Then these values (`extract_length()` and `window_size()`) can be calculated, and the user has easier numbers to enter.

An optional argument could also be the timestamp where the visualization should start.

Reply:
ooh I like this. Like, give me this full convo in 10 lines please; or, give me this convo in lines of 5 minutes. Yes that makes a lot of sense. The two variables `n_lines` and `line_duration` would be mutually exclusive then?

Thinking ahead of plotting, would it make sense to have the line duration of `n_lines` be some nearest reasonable round number?

re: timestamp, for true portability, I now think the `add_lines()` function should not bother with starts or end (see my [comment above](https://github.com/elpaco-escience/talkr/pull/10#discussion_r1159674955)) — it should just add `line`, `begin0`, `end0` cols to any data you feed it; the cutting or cropping should be done elsewhere

Reply:
BTW I also like the humanness of working with whole minutes, not milliseconds, even if under the hood we deal in ms. Still from working with transcripts I do anticipate that sub-minute lines are common (e.g., 10-20 seconds), so perhaps `line_duration_seconds` (seconds) and `line_duration_minutes`? Or `line_duration=10,unit="minutes"` with a sensible default? Trick is to keep the number of options managable and the defaults useful.